chemprop/nn/predictors.py

```diff
@@ -170,6 +170,9 @@ class MveFFN(RegressionFFN):
     n_targets = 2
     _T_default_criterion = MVELoss
 
+    def __post_init__(self):
+        self.ffn.append(nn.Unflatten(-1, (self.n_tasks, self.n_targets)))
```

Comment:
this is not a dataclass though?

Reply:
hmmm that is true, where would you suggest doing this append of Unflatten? It has to wait after the initialization of RegressionFFN, that's why I put it into the post init function.

Reply:
@KnathanM Good call! Fixed!